goson.go

```diff
@@ -112,6 +115,24 @@ func GenerateJSONSchema(jsonStr string) (MapString, error) {
 	return res, nil
 }
 
+func GenerateJSONSchemaFromURL(url string) (MapString, error) {
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	defer resp.Body.Close()
+	body, _ := ioutil.ReadAll(resp.Body)
+
+	return GenerateJSONSchema(string(body))
+}
+
+func GenerateJSONSchemaFromFile(ir io.Reader) (MapString, error) {
```

Comment:
Better to use the param is the path to the file, so we can just pass the path file.

Reply:
Then add a testfile.json to the repository, and make test against it

Reply:
Okay then, let me fix it